plugins/catalog/package.json

```diff
@@ -44,6 +44,7 @@
     "@backstage/plugin-catalog-react": "^1.1.1-next.0",
     "@backstage/plugin-search-common": "^0.3.4",
     "@backstage/plugin-search-react": "^0.2.0",
+    "@backstage/plugin-techdocs": "1.1.2-next.0",
```

Comment:
We should probably have a lint rule or build step that checks for this (\cc @rugvip), but it's very important that plugins don't directly depend on other plugins.

It looks like the only reason for this is the lowercase function? If so, a couple of options:

- Either just duplicate the logic in the catalog and don't worry about code duplication.
- Or move the function into `@backstage/plugin-techdocs-react`, which is intended up house cross-plugin, shared frontend code.

Reply:
Thanks, it was my main concern when I was importing the function, but maybe [my comment](https://github.com/backstage/backstage/pull/11527#issuecomment-1136037238) wasn't very descriptive.